src/Discord/Parts/Embed/Embed.php

```diff
@@ -38,7 +38,107 @@ class Embed extends Part
      * {@inheritdoc}
      */
     protected $fillable = ['title', 'type', 'description', 'url', 'timestamp', 'color', 'footer', 'image', 'thumbnail', 'video', 'provider', 'author', 'fields'];
-
+    
+    /**
+     * Constructs a new instance.
+     * @param array  $embed
+     * @throws \Throwable
+     */
+    function __construct(array $embed = array()) {
```

Comment:
Needs to implement [this constructor](https://github.com/teamreflex/DiscordPHP/blob/4a7a5218c9e13469cdb223f968a0774b2bfd0512/src/Discord/Parts/Part.php#L123) otherwise part won't work